lib/Ohmbrewer_Pump.cpp

```diff
@@ -61,6 +63,7 @@ int Ohmbrewer::Pump::doWork() {
  * @returns The time taken to run the method
  */
 int Ohmbrewer::Pump::doDisplay(Ohmbrewer::Screen *screen) {
+    screen->print("Pump");
```

Comment:
If you're going to delete the header, then do it, and to be consistent you should do the same for at least heating elements and probably sensors too.
However, I wouldn't (or at least I'd leave it to another PR).
Think of it this way: how many pieces of equipment can we have other than sensors? 6 or less, right? RIMS + 1 other, 2 thermostats? With the header, we can display all of those things. Now, we might say that we can wire all the sensors to one power pin (in addition to the bus) and so we get a stupid high number of temperatures on the display, but I'd argue that's more of an argument for refactoring the temperature display section than anything else (possibly displaying 2 readings per line, for example). Ditto if for some reason we can squeeze more relay based stuff under one heading.
If you're trying to handle the use case where we have just a single pump and something else, then I'd suggest not using the displayPumps method at all. Either make a separate displayPump method that appends that text in front so it's clear you're only interested in a single pump, or do some special stuff, like we do with the recirculation pump in RIMS.